src/OpenTelemetry.Instrumentation.Http/HttpClientInstrumentation.cs

```diff
@@ -22,10 +22,26 @@ namespace OpenTelemetry.Instrumentation.Http
     /// </summary>
     internal sealed class HttpClientInstrumentation : IDisposable
     {
+        private static readonly HashSet<string> DiagnosticSourceEventsNet7OrGreater = new()
```

Comment:
nit: Rename these HashSets to something like: `ExcludedEventsNet7OrGreater` and `ExcludedEvents`

Reply:
Done